flag_conformance_test.go

```diff
@@ -0,0 +1,22 @@
+package jsonnext_test
+
+import (
+	"flag"
+	"testing"
+
+	"foxygo.at/jsonnext"
+	"foxygo.at/jsonnext/conformance"
+)
+
+type suite struct{}
+
+func (s *suite) Parse(t *testing.T, args []string) (*jsonnext.Config, error) {
```

Comment:
same here - the term suite let me to think this is a standard testify suite, but really this is about being a flagParser, right?

Reply:
I get you, but testify does not get to claim the name "suite" all for itself, and this test file does not import testify at all. Again, I will ponder.